tests/manager.rs

```diff
@@ -82,7 +82,7 @@ async fn create_queue(suffix: &str) -> Queue {
     let stream = format!("stream{suffix}").into();
     let config = QueueConfig { stream };
     let client = redis::Client::open("redis://127.0.0.1/").expect("to create redis client");
-    let conn = client.get_tokio_connection_manager().await.expect("to get connection");
```

Comment:
why you changed it?

Reply:
get_tokio_connection_manager is now deprecated